docs/administration.platforms.md

```diff
@@ -91,13 +91,23 @@ If you want to remove quotas from a platform just click the "-" button on the ri
 
 ![Manage Platform Quota Definitions](assets/tenants/platform-quota-definitions.png)
 
-### Platform Deactivation
+### Deprecate Platform
 
-Via the "Danger Zone" tab you can deactivate or reactivate your meshPlatform.
+**Deprecation** will prevent the meshPlatform from being selected for new meshTenants. Existing meshTenants of this meshPlatform will still be replicated and can still be accessed via the meshPanel. Your users can recognize a meshTenant of a deprecated meshPlatform by a yellow bubble on the platform icon in the projects overview on the customer control plane. If your users are on the tenant control plane, a yellow status label will appear in the header of the control plane. You can also **undo the deprecation**. This allows your users to select the meshPlatform for new meshTenants again.
 
```

Comment:
```suggestion
**Deprecation** will prevent the meshPlatform from being selected for new meshTenants. Existing meshTenants of this meshPlatform will still be replicated and can still be accessed via the meshPanel. Your users can recognize a meshTenant of a deprecated meshPlatform by a yellow bubble on the platform icon in the "Projects Overview" screen on the Customer Control plane. If the user is on the tenant control plane, a yellow status label will appear in the header of the control plane. You can also **undo the deprecation**. This allows your users to select the meshPlatform for new meshTenants again.

```

Reply:
I could not find tenant control plane status ( e.g. depreciation) Could you help me with which screen it is?